exercises/arrays/allPositive/allPositive.js

```diff
@@ -11,13 +11,22 @@
  * @returns {boolean} True if every number in the input array is positive
  *   and false otherwise.
  */
+
+//  How do I make the User input the array?
 function allPositive(array) {
-  // This is your job. :)
-}
+  let result = true;
+  for (let i = 0; i < array.length; i++){
+    if (array[i] <= 0){
+      result = false;
+      break;
```

Comment:
Hmm, you must have some prior experience to use something like `break`!

When you use `return` the function will stop immediately regardless of how deeply nested the context is and return the given value to the caller. Here's another version that uses only `return`:


```js
function allPositive(array) {
  for (let i = 0; i < array.length; i++) {
    if (array[i] <= 0) {
      return false;
    }
  }

  return true;
}
```